markdown/preface.markdown

```diff
@@ -159,20 +160,24 @@ Although this book will focus on NumPy and SciPy,
 it is the many surrounding packages that make Python a powerhouse for
 scientific computing.
 
-
 ## The Great Cataclysm: Python 2 vs. Python 3
 
 In your Python travels, you may have already heard a few rumblings about which version of Python is better.
 You may have wondered why it's not just the latest version.
 
-At the end of 2008, the Python core developers released Python 3.0, a major update to Python with better Unicode (international) text handling, type consistency, and streaming data handling, among other improvements.
-To quote Douglas Adams in The Hitchhiker's Guide to the Galaxy, "this has made
-a lot of people very angry and been widely regarded as a bad move."
```

Comment:
That it was necessary remains in contention. But no one in their right mind would come out reading the preface thinking sticking to Python 2 is the right move! But *ignoring* all the text out there that *does* say you should stick to Python 2 is burying our heads in the sand, and might just leave readers confused. "Did these guys not read the post by X?", one might wonder, reading a pro-Python 3 preface that ignores all the criticism out there.